core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/diskRoutesModels.scala

```diff
@@ -42,7 +42,8 @@ final case class ListPersistentDiskResponse(id: DiskId,
                                             size: DiskSize,
                                             diskType: DiskType,
                                             blockSize: BlockSize,
-                                            labels: LabelMap
+                                            labels: LabelMap,
+                                            workspaceId: Option[WorkspaceId]
```

Comment:
Added optional `workspaceId` to the v1 `getDisk` and `listDisks` API responses. 

(It was already there for runtimes and apps)